src/main/java/io/jenkins/plugins/sprp/YAML_BranchProjectFactory.java

```diff
@@ -69,6 +97,7 @@ public String getScriptPath(){
     }
 
     @Override protected FlowDefinition createDefinition() {
+        System.out.println("Credential id = " + credentialsId);
```

Comment:
Please never print to STDOUT. If you want to log something, use `java.util.logging.Logger`

Reply:
I will keep that in mind in future, thanks :thumbsup: